recirq/quantum_chess/circuit_transformer_test.py

```diff
@@ -30,62 +30,67 @@
 d1 = cirq.NamedQubit('d1')
 
 
+@pytest.mark.parametrize('transformer',
+                         [ct.ConnectivityHeuristicCircuitTransformer,
+                          ct.DynamicLookAheadHeuristicCircuitTransformer])
 @pytest.mark.parametrize('device',
-                         (cirq.google.Sycamore23, cirq.google.Sycamore))
-def test_single_qubit_ops(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
+                         [cirq.google.Sycamore23, cirq.google.Sycamore])
+def test_single_qubit_ops(transformer, device):
     c = cirq.Circuit(cirq.X(a1), cirq.X(a2), cirq.X(a3))
-    transformer.qubit_mapping(c)
-    c = transformer.transform(c)
-    device.validate_circuit(c)
+    t = transformer(device)
+    device.validate_circuit(t.transform(c))
 
 
+@pytest.mark.parametrize('transformer',
+                         [ct.ConnectivityHeuristicCircuitTransformer,
+                          ct.DynamicLookAheadHeuristicCircuitTransformer])
 @pytest.mark.parametrize('device',
-                         (cirq.google.Sycamore23, cirq.google.Sycamore))
-def test_single_qubit_with_two_qubits(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
+                         [cirq.google.Sycamore23, cirq.google.Sycamore])
+def test_single_qubit_and_two_qubits_ops(transformer, device):
     c = cirq.Circuit(cirq.X(a1), cirq.X(a2), cirq.X(a3),
                      cirq.ISWAP(a3, a4) ** 0.5)
-    transformer.qubit_mapping(c)
-    device.validate_circuit(transformer.transform(c))
+    t = transformer(device)
+    device.validate_circuit(t.transform(c))
 
 
+@pytest.mark.parametrize('transformer',
+                         [ct.ConnectivityHeuristicCircuitTransformer,
+                          ct.DynamicLookAheadHeuristicCircuitTransformer])
 @pytest.mark.parametrize('device',
-                         (cirq.google.Sycamore23, cirq.google.Sycamore))
-def test_three_split_moves(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
+                         [cirq.google.Sycamore23, cirq.google.Sycamore])
+def test_three_split_moves(transformer, device):
     c = cirq.Circuit(qm.split_move(a1, a2, b1), qm.split_move(a2, a3, b3),
                      qm.split_move(b1, c1, c2))
-    transformer.qubit_mapping(c)
-    device.validate_circuit(transformer.transform(c))
+    t = transformer(device)
+    device.validate_circuit(t.transform(c))
 
 
+@pytest.mark.parametrize('transformer', [ct.ConnectivityHeuristicCircuitTransformer])
```

Comment:
Why are we parameterizing if it only has one option?  Same with several functions below.

Reply:
added `DynamicLookAheadHeuristicCircuitTransformer` to the list now that it's updated to use `SwapUpdater`.